pkg/templates/manifest.go

```diff
@@ -138,6 +146,12 @@ data:
       secret:
         secretName: ssh-key
         defaultMode: 256
+{{- end}}
+{{- if .CustomRegistries }}
+    extra-volumes:
```

Comment:
Both this and the ssh logic above add this line if needed. I'll need to make sure it doesnt get duplicated.